src/MissionManager/MavCmdInfo.json

```diff
@@ -588,7 +588,20 @@
         { "id": 2500, "rawName": "MAV_CMD_VIDEO_START_CAPTURE", "friendlyName": "MAV_CMD_VIDEO_START_CAPTURE" },
         { "id": 2501, "rawName": "MAV_CMD_VIDEO_STOP_CAPTURE", "friendlyName": "MAV_CMD_VIDEO_STOP_CAPTURE" },
         { "id": 2800, "rawName": "MAV_CMD_PANORAMA_CREATE", "friendlyName": "MAV_CMD_PANORAMA_CREATE" },
-        { "id": 3000, "rawName": "MAV_CMD_DO_VTOL_TRANSITION", "friendlyName": "MAV_CMD_DO_VTOL_TRANSITION" },
+        {
+            "id": 3000,
+            "rawName": "MAV_CMD_DO_VTOL_TRANSITION",
+            "friendlyName": "VTOL Transition",
+            "description":  "Perform flight mode transition",
+            "category":     "Basic",
+            "param7": {
+                "label":            "Mode:",
+                "default":          0,
```

Comment:
You need to start out with a valid mission command though. 0 is undefined  in MAV_VTOL_STATE which is likely going to cause problem for other firmwares. If "Do nothing is an option" then there needs to be a "Do nothing" enum value. The fact that MAV_VTOL_STATE enum seems to have enums for both the transition command as well as I'm assuming another command which queries state is not great. Those two things should really be in separate enums.